libs/cart/src/models/cart.ts

```diff
@@ -17,18 +22,45 @@ export interface DaffCart extends DaffIdentifiable {
 	 * @deprecated use totals instead
 	 */
   grand_total: number;
+  /**
+   * A list of coupon applied to the cart.
+   */
   coupons: DaffCartCoupon[];
+  /**
+   * A list of cart items.
+   */
   items?: DaffCartItem[];
+  /**
+   * The billing address of the cart.
+   */
   billing_address: DaffCartAddress | null;
+  /**
+   * The shipping address of the cart.
+   */
   shipping_address: DaffCartAddress | null;
+  /**
+   * The selected payment method of the cart.
+   */
   payment?: DaffCartPaymentMethod;
+  /**
+   * A list of totals for the cart.
+   */
   totals: DaffCartTotal[];
+  /**
+   * The selected shipping method.
+   */
   shipping_information: DaffCartShippingInformation | null;
+  /**
+   * A list of possible shipping methods.
+   */
   available_shipping_methods: DaffCartShippingRate[];
+  /**
+   * A list of possible payment methods.
```

Comment:
Even though it would be using the same word, I think "A list of available payment methods" sounds better

Reply:
changed